### PR TITLE
Extra tweaks for ExamplePlugin

### PR DESCRIPTION
**Pull request by Frago9876543210**
- `.gitattributes` now uses LF for code (useful when many people work on same plugin)
- bit changes in `composer.json` about versions
- fixed cs (should we take phpcs-fixer from pmmp/PocketMine-MP?)

### CODE REVIEW

File: phpstan/composer.json [hunk 1]
```diff
@@ -1,14 +1,19 @@
 {
    "name": "pocketmine/phpstan-plugin-config",
    "require": {
-      "phpstan/phpstan": "^1.2.0",
-      "pocketmine/pocketmine-mp": "4.0.0",
+      "phpstan/phpstan": "^1.0",
+      "pocketmine/pocketmine-mp": "^4.0",
```
Comment (dktapps) [inline phpstan/composer.json:5]: It's supposed to be anchored to 4.0.0.

File: phpstan/composer.json [hunk 2]
```diff
@@ -1,14 +1,19 @@
 {
    "name": "pocketmine/phpstan-plugin-config",
    "require": {
-      "phpstan/phpstan": "^1.2.0",
-      "pocketmine/pocketmine-mp": "4.0.0",
+      "phpstan/phpstan": "^1.0",
```
Comment (dktapps) [inline phpstan/composer.json:4]: phpstan is too volatile to use a version constraint this wide.

File: .gitattributes [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Auto detect text files and perform LF normalization
 * text=auto
+*.php text eol=lf
+*.sh text eol=lf
+*.txt text eol=lf
+*.properties text eol=lf
+*.neon text eol=lf
+*.bat text eol=crlf
+*.cmd text eol=crlf
+*.ps1 text eol=crlf
```
Comment (dktapps) [inline .gitattributes:10]: These changes seem mostly redundant. Only .php and .neon apply here.

**Review by dktapps — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 3 above)
Comment (Frago9876543210) [reply .gitattributes:10]: what if you want add setup/build scripts to plugin repo? `*.properties` and all things that doesn't related to automation can be removed

Comment (dktapps) [reply .gitattributes:10]: "what if" is not a good reason to make a change. If we decide to do such things in the future, the supporting changes for gitattributes would be added at the appropriate time.

**Pull request merged**